AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_tools_utils.py

```diff
@@ -49,6 +53,29 @@ def compare_colors(color1: azlmbr.math.Color, color2: azlmbr.math.Color, buffer:
     )
 
 
+def verify_one_material_document_opened(
+        material_document_ids_list: [azlmbr.math.Uuid], opened_document_index: int) -> bool:
+    """
+    Validation helper to verify if the document at opened_document_index value in the material_document_ids list
+    is the only opened document.
+    Returns True on success, False on failure.
+
+    :param material_document_ids_list: List of material document IDs used for making the document opened check.
+    :param opened_document_index: Index number of the one material document that should be open
+    :return: bool
+    """
+    if not is_document_open(material_document_ids_list[opened_document_index]):
+        return is_document_open(material_document_ids_list[opened_document_index])
+
+    material_document_ids_verification_list = material_document_ids_list.copy()
+    material_document_ids_verification_list.pop(opened_document_index)
+    for closed_material_document_id in material_document_ids_verification_list:
+        if is_document_open(closed_material_document_id):
+            return is_document_open(closed_material_document_id)
```

Comment:
doesn't this return True when a document that should be closed is open? shouldn't this return False overall that an expected closed document was actually open? again maybe just return a static bool here False

Reply:
true, i updated it to return `False` same for the other comment